### PR TITLE
Add order book page

### PR DESCRIPTION
**Pull request by anxolin**
Part of #791

I would like to create as mentioned in #791  a placeholder so Felix can work on integrating his order book.

his PR includes:
- A new page with the selectors logic
- A new component for the Otherbook
- Add a wrapper with some example CSS --> just to show how to add CSS, but Felix pls remove the CSS in the next PR

image

Test it by navigating to `/book` address:
* http://localhost:8080/book
* https://pr800--dexreact.review.gnosisdev.com/book

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr800--dexreact.review.gnosisdev.com

File: src/pages/OrderBook.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import TokenSelector from 'components/TokenSelector'
+import { useTokenList } from 'hooks/useTokenList'
+import { useWalletConnection } from 'hooks/useWalletConnection'
+import { Network } from 'types'
+import useSafeState from 'hooks/useSafeState'
+import { TokenDetails } from 'types'
+
+const OrderBook: React.FC = () => {
+  const { networkId } = useWalletConnection()
+  const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
```
Comment (alfetopito) [inline src/pages/OrderBook.tsx]: Do we need to show prices, or even, does the tool have prices for anything that's not Mainnet?

**Comment by fleupold** [comment]
Should I add the code to this PR or create a new one on top?

File: src/pages/OrderBook.tsx (hunk 1 above)
Comment (anxolin) [reply src/pages/OrderBook.tsx:13]: First version Felix did was using pure web3, so only that was needed (that, and the batchexchange contract and the viewer)

I think he is using now his own service, and I believe he also has two environments (rinkeby and mainnet), so probably yes, I think we should show the current network, and mainnet as a default

**Comment by anxolin** [comment]
> Should I add the code to this PR or create a new one on top?

One on top is better

File: src/pages/OrderBook.tsx (hunk 1 above)
Comment (fleupold) [reply src/pages/OrderBook.tsx:13]: Yes, I'm querying the backend service and need network info to form the correct URL

**Comment by c3rnst** [comment]
cool! from where will the order book be accessible on the interface? I have some concerns that it will increase confusion on the site: first we say it's not one order book per pair and then we present it :) but in any case, good to have the info.

**Comment by anxolin** [comment]
> cool! from where will the order book be accessible on the interface? I have some concerns that it will increase confusion on the site: first we say it's not one order book per pair and then we present it :) but in any case, good to have the info.

@c3rnst  This page won't be visible probably (navigable). This is just for isolating a component Felix will work on. 

This component might be shown for example, as a popup in the trading widget if the user click in one suggested price or a link we make for showing the order book. 
Would that work?

**Comment by c3rnst** [comment]
> Would that work?

**Comment by anxolin** [comment]
@fleupold I've merged this one. Point yours to develop if you want

**Pull request merged**